numpy/core/include/numpy/ufuncobject.h

```diff
@@ -251,15 +251,21 @@ typedef struct _tagPyUFuncObject {
 #endif
 
 /*
+ * UFunc has unit of 0, and the order of operations can be reordered
+ * This case allows reduction with multiple axes at once.
+ */
+#define PyUFunc_Zero 0
+/*
  * UFunc has unit of 1, and the order of operations can be reordered
  * This case allows reduction with multiple axes at once.
  */
 #define PyUFunc_One 1
 /*
- * UFunc has unit of 0, and the order of operations can be reordered
- * This case allows reduction with multiple axes at once.
+ * UFunc has unit of -1, and the order of operations can be reordered
+ * This case allows reduction with multiple axes at once. Intended for
```

Comment:
There might be others, but I could not think of any. Hence the bit of explanation as to why this otherwise useless macro is there. I suppose it could be "Introduced" instead of "Intended".